src/plugins/data-card/components/case-attribute.tsx

```diff
@@ -65,19 +65,14 @@ export const CaseAttribute: React.FC<IProps> = observer(props => {
   const editingLabel = currEditFacet === "name" && currEditAttrId === attrKey;
   const editingValue = currEditFacet === "value" && currEditAttrId === attrKey;
 
-  const validCompletions = useCallback((aValues: string[], prefixString: string) => {
-    const prefixStringLC = prefixString.toLowerCase();
-    const values = uniq(aValues).sort();
-    if (editingValue && valueCandidate.length > 0){
-      return values.filter((value) => {
-        return value && typeof(value)==='string' && !isImageUrl(value)
-               && value.toLowerCase().startsWith(prefixStringLC);
-      }) as string[];
-    } else {
-      return values.filter((value) => {
-        return value && typeof(value)==='string' && !isImageUrl(value);
-      }) as string[];
-    }
+  const validCompletions = useCallback((aValues: string[], userString: string) => {
+    const values = uniq(aValues).filter(value => typeof value === 'string').sort();
+    const regex = new RegExp(`${userString}`, 'i');
```

Comment:
You need to escape the `userString` before creating a regex from it, e.g. using `escapeStringRegexp` from the "escape-string-regexp" library.

Reply:
ah, thank you